<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editclasswidget</class>
 <widget class="QWidget" name="editclasswidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>433</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>11</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>添加课程</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_class_name_line">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>52</y>
      <width>256</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>课程名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>52</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/classicon.svg</normaloff>:/Themes/Assets/classicon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changecolorbutton">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>96</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>点此更改颜色</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/changecolor_icon.svg</normaloff>:/Themes/Assets/changecolor_icon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="extro_info_editline">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>134</y>
      <width>256</width>
      <height>37</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>备注 ( 可不填 ) </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>134</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/extra_info_button.svg</normaloff>:/Themes/Assets/extra_info_button.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>178</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>193</y>
      <width>59</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>223</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/canlendar.svg</normaloff>:/Themes/Assets/canlendar.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_week_style">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>223</y>
      <width>256</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>269</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/clock_icon.svg</normaloff>:/Themes/Assets/clock_icon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_time_style">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>269</y>
      <width>256</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_51">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>396</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

    background-color: transparent;
    padding: 5px 5px; /* 内边距 */
    font-size: 13px; /* 字体大小 */
    border-radius: 5px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgba(0,0, 0, 220);
}

QPushButton:pressed {
    background-color:  rgba(0, 0, 0,220);
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>396</y>
      <width>94</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #00CED1; /* 按钮背景颜色为蓝绿色 */
    color: white; /* 按钮文字颜色为白色 */
    border: none; /* 无边框 */
    padding: 5px 5px; /* 内边距 */
    font-size: 13px; /* 字体大小 */
    border-radius: 5px; /* 圆角 */
}

QPushButton:hover {
    background-color: #00B2B2; /* 鼠标悬停时背景颜色变深 */
}

QPushButton:pressed {
    background-color: #008C8C; /* 鼠标按下时背景颜色更深 */
}</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>96</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>█</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_class">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>9</y>
      <width>38</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/Themes/Assets/delete.svg</normaloff>:/Themes/Assets/delete.svg</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
